apps/utils/scripts/README.md

```diff
@@ -271,6 +269,14 @@ run-bfd-pipeline [options] run_mode
 -Z                        Enable debugger support and wait for debugger to connect.
 -h                        Prints this help message.
 
+Supported long options:
+
+--bene-load-options threads:batch_size:queue_multiple
+  Sets the RifLoader settings to use for beneficiary data.
+
+--claim-load-options: threads:batch_size:queue_multiple
```

Comment:
```suggestion
--claim-load-options threads:batch_size:queue_multiple
```

Reply:
Good catch.  Will fix here and in the script itself.